setup.py

```diff
@@ -15,7 +15,7 @@
 # not to append tensorflow and tensorflow_probability to the requirements:
 if os.environ.get("READTHEDOCS") != "True":
     requirements = [
-        "tensorflow>=2.2.0",
+        "tensorflow>=2.5.0",
```

Comment:
This would mean GPflow will stop supporting TF 2.2 - 2.4 -- can we not find a workaround to support those as well?

Reply:
Yep, your other comment achieved that.